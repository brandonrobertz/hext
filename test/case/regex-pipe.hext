<a href|filter(/([]-z\-0-9]*)$/)=>user_id
   @text|trim=>user_name
/>
