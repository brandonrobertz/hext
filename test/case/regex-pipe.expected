{"user_id":"alice-23","user_name":"Alice"}
{"user_id":"bob-22","user_name":"Bob"}
{"user_id":"tom-24","user_name":"Tom"}
